Dockerfile

```diff
@@ -16,11 +16,11 @@ FROM alpine:latest
 
 MAINTAINER GoCD <go-cd-dev@googlegroups.com>
 
-LABEL gocd.version="17.8.0" \
+LABEL gocd.version="17.9.0" \
```

Comment:
No need to check this in I believe. The images will be built from the right GoCD version and the Dockerfile will be updated at the time of the release.

Reply:
Bad commit. Removed the file